hott-a4-exercises.tex

```diff
@@ -1,7 +1,7 @@
 % -*- mode: LaTeX -*-
 % This is a main LaTeX file for generating the HoTT book
-% for printing on A4 paper. The links are black in this
-% version.
+% for printing on A4 paper, with exercises on per page.
```

Comment:
Another typo: "on per page" should be "one per page", or maybe "printed one per page" to match what you wrote it `hott-letter-exercises`.

Reply:
Fixed in 9386c1e